common/src/main/scala/explore/common/ScienceQueries.scala

```diff
@@ -124,9 +128,11 @@ object ScienceQueries {
       extends AnyVal {
     def toInput: GmosNorthLongSlitAdvancedConfigInput =
       GmosNorthLongSlitAdvancedConfigInput(
+        Input.ignore,
```

Comment:
There are some new fields in the `*AdvancedConfigInput`s. I'm just ignoring them for now.